eventhub/endpoint.bal

```diff
@@ -280,24 +285,24 @@ public client class Client {
 
     // Publisher Client Operations
 
-    # Send a single event
+    # Send a single event to an Event Hub.
     #
-    # + eventHubPath - event hub path
-    # + data - event data
-    # + userProperties - user properties
-    # + brokerProperties - broker properties
-    # + partitionId - partition ID
-    # + publisherId - publisher ID 
-    # + partitionKey - partition Key
-    # + return - @error if remote API is unreachable
+    # + eventHubPath - Event Hub path (Event Hub name)
+    # + data - Event data in string|xml|json|byte[] format
+    # + userProperties - Map of custom properties
+    # + brokerProperties - Map of broker properties
+    # + partitionId - Partition Id
+    # + publisherId - Publisher Id 
+    # + partitionKey - Partition Key
+    # + return - Nil on success, else returns an error if remote API is unreachable
     @display {label: "Send an Event"}
-    remote isolated function send(@display {label: "Event hub path"} string eventHubPath, 
-                                  @display {label: "Event data"} 
+    remote isolated function send(@display {label: "Event Hub Path"} string eventHubPath, 
+                                  @display {label: "Event Data"} 
                                   string|xml|json|byte[]|mime:Entity[]|stream<byte[], io:Error> data, 
-                                  @display {label: "User properties (Optional)"} map<string>? userProperties = (), 
-                                  @display {label: "Broker properties (Optional)"} map<anydata>? brokerProperties = (), 
-                                  @display {label: "Partition ID (Optional)"} int? partitionId = (), 
-                                  @display {label: "Publisher ID (Optional)"} string? publisherId = (), 
+                                  @display {label: "User Properties (Optional)"} map<string>? userProperties = (), 
```

Comment:
No need to mention `(Optional)` on the label.

Reply:
Resolved in 4de8b34